### PR TITLE
company id issue resolved while creating the companies

### PR DESCRIPTION
**Pull request by Uttkarsh-raj**
While creating a company we are getting an issue of company_id is not unique which is because it is not getting set, this will give Integrity error in the code while making the requests.

Previously:

Screenshot (99)

Now:

Screenshot (103)

### CODE REVIEW

**Comment by AtmegaBuzz** [comment]
LGTM!

File: website/models.py [hunk 1]
```diff
@@ -63,6 +63,11 @@ class Company(models.Model):
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
+        if not self.company_id:
```
Comment (arkid15r) [inline website/models.py:67]: Let's avoid the empty string case:
```suggestion
        if self.company_id is None:
```

**Review by arkid15r — COMMENTED** [review]
I think I should have mentioned this during the previous PR review: Django has a separate [UUID Field](https://docs.djangoproject.com/en/5.0/ref/models/fields/#uuidfield). The `company_id` should probably be `company_uuid` or just `uuid` if possible as it already is inside of `Company` context.

**Comment by Uttkarsh-raj** [comment]
> I think I should have mentioned this during the previous PR review: Django has a separate [UUID Field](https://docs.djangoproject.com/en/5.0/ref/models/fields/#uuidfield). The `company_id` should probably be `company_uuid` or just `uuid` if possible as it already is inside of `Company` context.

Or another way i thought of doing this was in the company_id field give the default value as uuid.uuid4 this also did the task but it could have generated some issue with respect to data already present in the database..

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
Can we please use the auto id built in?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
We can remove company_id it's redundant

**Comment by Uttkarsh-raj** [comment]
> We can remove company_id it's redundant

Ok will make this change...

**Review by DonnieBLT — CHANGES_REQUESTED** [review]

**Review by DonnieBLT — APPROVED** [review]

File: website/models.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ class Company(models.Model):
     managers = models.ManyToManyField(User, related_name="user_companies")
     name = models.CharField(max_length=255)
     logo = models.ImageField(upload_to="company_logos", null=True, blank=True)
-    company_id = models.CharField(max_length=255, unique=True, editable=False)  # uuid
+    # company_id = models.CharField(max_length=255, unique=True, editable=False)  # uuid
```
[the remark this answers is not in the document]
Comment (Uttkarsh-raj) [reply website/models.py]: got changed when i pulled... should have checked carefully... thank you...

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**